xdsl/dialects/arith.py

```diff
@@ -765,13 +768,16 @@ def parse(cls, parser: Parser):
         operand1 = parser.parse_unresolved_operand()
         parser.parse_punctuation(",")
         operand2 = parser.parse_unresolved_operand()
+        fastmath = None
+        if parser.parse_optional_keyword("fastmath") is not None:
+            fastmath = FastMathFlagsAttr(FastMathFlagsAttr.parse_parameter(parser))
```

Comment:
```suggestion
        fastmath = None if parser.parse_optional_keyword("fastmath") is None else FastMathFlagsAttr(FastMathFlagsAttr.parse_parameter(parser))
```

Reply:
Hmm, I'm not sure I prefer your suggestion TBH, I'd maybe suggest moving the `fastmath = None` into an `else` clause but otherwise I think it's OK to split into two lines.